src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/PersonServiceIntTest.kt

```diff
@@ -0,0 +1,154 @@
+package uk.gov.justice.digital.hmpps.personrecord.service
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.justice.digital.hmpps.personrecord.integration.MessagingMultiNodeTestBase
+import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.PersonEntity
+import uk.gov.justice.digital.hmpps.personrecord.model.identifiers.CROIdentifier
+import uk.gov.justice.digital.hmpps.personrecord.model.identifiers.PNCIdentifier
+import uk.gov.justice.digital.hmpps.personrecord.model.person.OtherIdentifiers
+import uk.gov.justice.digital.hmpps.personrecord.model.person.Person
+import uk.gov.justice.digital.hmpps.personrecord.model.types.SourceSystemType
+
+class PersonServiceIntTest : MessagingMultiNodeTestBase() {
+
+  @Autowired
+  private lateinit var personService: PersonService
+
+  @BeforeEach
+  override fun beforeEach() {
+    super.beforeEach()
+    // need to delete as obscure fields to search on???
+    personRepository.deleteAll()
```

Comment:
I know it's quite a lot (especially with the various identifiers), but I think we should consider better random generation of data as an alternative to this.

Reply:
I kept it for reasoning above, happy to chat about.